{
  "Header": {
    "Original": "CVE-2020-35533 (In LibRaw, an out-of-bounds read vulnerability exists within the \"LibR ...)",
    "Line": 151189,
    "ID": "CVE-2020-35533",
    "Description": "(In LibRaw, an out-of-bounds read vulnerability exists within the \"LibR ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-3113-1}",
      "Line": 151190,
      "Type": "xref",
      "Bugs": [
        "DLA-3113-1"
      ]
    },
    {
      "Original": "- libraw 0.20.0-4",
      "Line": 151191,
      "Type": "package",
      "Package": "libraw",
      "Kind": "fixed",
      "Version": "0.20.0-4"
    },
    {
      "Original": "NOTE: https://github.com/LibRaw/LibRaw/commit/a6937d4046a7c4742b683a04c8564605fd9be4fb (0.20-RC2)",
      "Line": 151192,
      "Type": "NOTE",
      "Description": "https://github.com/LibRaw/LibRaw/commit/a6937d4046a7c4742b683a04c8564605fd9be4fb (0.20-RC2)"
    },
    {
      "Original": "NOTE: https://github.com/LibRaw/LibRaw/issues/273",
      "Line": 151193,
      "Type": "NOTE",
      "Description": "https://github.com/LibRaw/LibRaw/issues/273"
    }
  ]
}